Automattic/woocommerce-payments
# feat: add logging setting

Fixes https://github.com/Automattic/woocommerce-payments/issues/1836

#### Changes proposed in this Pull Request









![Markup 2021-05-26 at 16 15 45](https://user-images.githubusercontent.com/273592/119732327-a6768600-be3d-11eb-9ca9-888d2102a769.png)


#### Testing instructions





- Ensure you have the "grouped settings" flag enabled in WCPay Dev Tools
- Go to the WCPay settings: http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout
- Logging mode checkbox should be checked/unchecked/disabled
- Field behaves in the same way the checkbox also behaves in the old settings view

-------------------
